standalone/porkcron.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env sh
+
+# Move to /etc/porkcron
+cd /etc/porkcron
+
+# Read the .env file if it exists
+if [ -e .env ]
+then
+  while IFS== read -r key value; do
+    case $key in
+      ''|\ \#*|\#*) continue ;;
+    esac
+    case $value in
+      \ \#*|\#*) continue ;;
+    esac
+    printf -v "$key" %s "$value"
+    export "$key"
+  done <.env
+else
+  echo "Could not find $PWD/.env to fetch the latest SSL certs!" 1>&2
+  echo "Trying anyway, it should work if the correct environment variables are set elsewhere." 1>&2
+fi
```

Comment:
Would a simple
```
export $(grep -v '^#' .env | xargs)
```
work instead? I wouldn't worry about corner cases, our .env file is super simple

Reply:
I could certainly test it with a clean .env file. Went with this because I needed something to counteract my own stupidity when testing and setting up

Reply:
Update: no it does not. `export` is pretty finicky here and as a shell builtin it also can't be called through `grep ... | xargs export`
```
gitserver@GiteaPi:~/porkcron$ export $(grep -v '^#' .env.example | xargs)
-bash: export: `#': not a valid identifier
-bash: export: `comma-separated': not a valid identifier
gitserver@GiteaPi:~/porkcron$ export $(grep -v '^\#' .env.example | xargs)
-bash: export: `#': not a valid identifier
-bash: export: `comma-separated': not a valid identifier
```

Reply:
Ended with almost the suggested solution. Allowing for whitespace before the comment marker, and using a simple sed call to fix unexpected results if any value contains `, ` by shaving off the space.
Purely as an accident, this does allow using spaces for e.g. friendlier softwrapping in editors